onnxruntime/core/framework/session_state.h

```diff
@@ -23,6 +23,7 @@
 #include "core/framework/node_index_info.h"
 #include "core/graph/graph_viewer.h"
 #include "core/framework/fuse_nodes_funcs.h"
+#include "core/session/onnxruntime_c_api.h"
 
```

Comment:
This seems like a structural issue due to defining OrtDeleter in the C api instead of having an opaque type there.

Reply:
Do we still need to #include the C API header?